### PR TITLE
Text-editor overflow hidden on iPad

### LINKED ISSUE #9104: Text-editor overflow hidden on iPad
#### Challenge name: Turn an Image into a link

https://www.freecodecamp.com/challenges/turn-an-image-into-a-link

#### Issue Description

I am using an iPad.
The bottom of the text editor and content is cut-off.
The text editor does not scale with the amount of text it contains. Nor does it scroll. This makes it impossible to edit the below-the-fold content. Although this is an issue on other challenges the above challenge link and name is a good example. This is the first challenge that this bug stops you from completing the challenge.

Please see below device information to reproduce issue.
#### Browser Information

* Browser Name, Version: Safari
* Operating System: iOS 9.3.2
* Mobile, Desktop, or Tablet: iPad 2

#### Screenshot

image

### PR DESCRIPTION
**Pull request by craigstroman**
#### Type of Change

- [X] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)
#### Checklist:

- [X] Tested changes locally.
- [X] Closes currently open issue (replace XXXX with an issue no): Closes #9104 
#### Description

I added some CSS to client/less/main.less near the bottom of the file.  It makes it so you can scroll to see the overflow text on the iPad.  I tested it in the iOS simulator and seems to fix the issue.  Although I was kind of confused about how they wanted it fixed.  If they would rather the height of the text editor change for the iPad.  I can change it to that also if it's preferred, please let me know.

### CODE REVIEW

File: client/less/main.less [hunk 1]
```diff
@@ -1175,6 +1175,11 @@ and (max-width : 400px)  {
   font-size: 17px;
 }
 
+// Challange editor CSS for iOS.  Issue 9104.
```
Comment (raisedadead) [inline client/less/main.less]: remove this comment line.

**Comment by raisedadead** [comment]
@craigstroman Thanks a lot for the pull request.

As camper-bot mentions, please amend your commit and remove the issue no.
```
git commit --amend
CHANGE the message
git push origin <pr branch name> --force
```

Also,

Edit the PR title on the top of this page and edit and remove the issue no., it should just be in the PR description.

**Comment by craigstroman** [comment]
Hi, 

Sorry about that.  I was kind of confused about the contributing instructions.

I amended my message and removed the text you requested be removed from the title.

Please let me know if there is anything else.

Thank you,
Craig

File: client/less/main.less (hunk 1 above)
Comment (atjonathan) [inline client/less/main.less]: This still doesn't seem to be removed.

Comment (craigstroman) [inline client/less/main.less]: Hi,

I wasn't sure exactly what was wanted.

Do you want the editor to expand on the iPad and show the text or it just to be scrollable?

I can change it so it's that way also.  Just that will take a bit longer.

Thank you,
Craig

Comment (raisedadead) [inline client/less/main.less]: @craigstroman you just need to remove the comment line

`// Challange editor CSS for iOS.  Issue 9104.`

this is not necessary.

Comment (craigstroman) [inline client/less/main.less]: I see, I will fix that also.

**Comment by craigstroman** [comment]
Hi,

I removed that comment from the CSS.    Please let me know if anything else should be fixed.

Thank you,
Craig

**Comment by raisedadead** [comment]
@FreeCodeCamp/issue-moderators bump for a QA and merge from anyone with a iPad.

**Comment by raisedadead** [comment]
Blocking for now, we are rolling out react infrastructure, and this will have to wait, apologies for the delay.

**Comment by raisedadead** [comment]
Blocking this as will have to take this change after we have the react editor in place.

**Comment by atjonathan** [comment]
#### **LGTM :+1:**
Thanks for contributing to the most awesome open source community :tada:

**Pull request merged**